eng/native/init-os-and-arch.sh

```diff
@@ -56,9 +56,11 @@ case "$CPUName" in
         if (NAME=""; . /etc/os-release; test "$NAME" = "Tizen"); then
             __BuildArch=armel
             __HostArch=armel
```

Comment:
nit: `__BuildArch` and `__HostArch` here and below are redundant and can be removed. We are overwriting them unconditionally in `build-commons.sh`.

Reply:
Thank you!